apps/server/src/config.ts

```diff
@@ -2,52 +2,71 @@ const processEnv = {}
 const envs = { ...process.env, ...(process as Record<string, any>).client_envs }
 
 Object.keys(envs).forEach((k) => {
-  (processEnv as Record<string, any>)[k.toUpperCase()] = envs[k]
+  ;(processEnv as Record<string, any>)[k.toUpperCase()] = envs[k]
```

Comment:
Is that semi-colon a typo?

Reply:
Prettier is doing it, and I don't know why